MPF.Processors/ProcessingTool.cs

```diff
@@ -372,6 +372,91 @@ static int ReadFromArrayBigEndian(byte[]? bytes, int offset)
             };
         }
 
+        /// <summary>
+        /// Determine the region based on the PlayStation serial code
+        /// </summary>
+        /// <param name="serial">PlayStation serial code</param>
+        /// <returns>Region mapped from name, if possible</returns>
+        public static Region? GetPlayStationRegion(string? serial)
```

Comment:
Minor nitpick: Please move this above `GetXGDRegion`

Reply:
Done. For alphabetical order reasons?